### PR TITLE
docs: fix typo in README.md and docs related to environments supported

### PR DESCRIPTION
**Pull request by ThisIsMissEm**
This is follow up to #501, branching from #503

### CODE REVIEW

File: docs/api/source/index.rst [hunk 1]
```diff
@@ -52,22 +52,32 @@ See `the release notes
...
+e.g. ``Map``, ``Set``, ``Promise``, ``Headers``, ``Array.prototype.includes``,
 ``Object.entries`` and ``String.prototype.endsWith``.
 
 Additionally, when using this package in an environment other than Node.js, you
 will need `a polyfill for Node's buffer module
 <https://www.npmjs.com/package/buffer>`__.
 
 Node.js Support
-~~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^^
```
Comment (chelseapinka) [inline docs/api/source/index.rst:80]: why update only this underline and not the supported environments?

**Review by chelseapinka — DISMISSED** [review]

File: src/notification.test.ts [hunk 2]
```diff
@@ -31,7 +31,7 @@ import {
 
 import crossFetch, { Response } from "cross-fetch";
 import type * as SolidClient from "@inrupt/solid-client";
-import type * as SolidClientAuthnBrowser from "@inrupt/solid-client-authn-browser";
+// import type * as SolidClientAuthnBrowser from "@inrupt/solid-client-authn-browser";
```
Comment (chelseapinka) [inline src/notification.test.ts:34]: do we want to leave this commented out or take it out all together?

Comment (ThisIsMissEm) [reply src/notification.test.ts:34]: Yeah, as this is inline with the rest of the code relating to this in this file.

Comment (ThisIsMissEm) [reply src/notification.test.ts:34]: See #307 for details of what was going on.. I can explain this more on slack or something.

**Review by NSeydoux — APPROVED** [review]

**Pull request merged**